bin/blackbox_view

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+#
+# blackbox_edit.sh -- Decrypt a file temporarily for edition, then re-encrypts it again
```

Comment:
It is not different.  I was simply unaware of blackbox_cat since this was not included in the README.  blackbox_cat is exactly what I was looking for, so I have removed blackbox_view.